modules/appeals_api/app/services/appeals_api/supporting_evidence/evidence_uploader.rb

```diff
@@ -15,7 +17,11 @@ def initialize(appeal, document, type:)
       def process!
         generate_evidence_submission!
         uploader.store!(document)
-        update_submission!
+        update_submission!('submitted')
+      rescue => e
+        log_message_to_sentry('Error saving to S3', :warning, error: e.to_s)
+        update_submission!('s3_error')
```

Comment:
I've actually found (while dealing with HLR errors) that having the error code & details stored with the model has helped greatly in triaging. What do you think about adding `code` and `details` to the EvidenceSubmissions model that we can stuff with these exception details as well as sending to sentry?

Even if we don't go that direction, I think the status should just be `error` since we'll use sentry to track down the problem with that GUID

Reply:
👀